ocs_ci/ocs/ui/views.py

```diff
@@ -46,6 +46,7 @@
     ),
     "search_operator_installed": ('input[data-test-id="item-filter"]', By.CSS_SELECTOR),
     "thin_sc": ('a[id="thin-link"]', By.CSS_SELECTOR),
+    "thin-csi_sc": ("//span[text()='(default) | csi.vsphere.vmware.com']", By.XPATH),
```

Comment:
Why we don't use ID locator here?

And:
Don't we need to add it also to add_capacity here:
https://github.com/red-hat-storage/ocs-ci/blob/d97ce8afad7a9c0f8f651d253add35b3f5235720/ocs_ci/ocs/ui/views.py#L749

Reply:
the development team changed the locator.
For `add capacity`, we are using the default sc... [without clicking on sc buttom]